.github/workflows/main.yml

```diff
@@ -5,20 +5,20 @@ on: [push]
 jobs:
   test-node:
     runs-on: ubuntu-latest
-    services:
-      mongodb:
-        image: mongo:3.6
-        ports:
-          - 27017:27017
     strategy:
       matrix:
         node-version: [10.x, 12.x, 14.x]
+        mongodb-version: ['3.6', '4.0', '4.2']
```

Comment:
@davidlehn I believe you added this.  As I mentioned before, there is no expectation that this is backwardly compatible with 3.x, and I don't think the index field length thing landed until v4.2.  I wrote all that up somewhere.  At any rate, it is my assertion that we only need to test here with 4.2.

Reply:
Yes, I added the matrix since it was only testing with 3.6 before: https://github.com/digitalbazaar/bedrock-mongodb/pull/38.

I had been thinking that this module might still work with 3.6.  But other modules might be using the newer features.  Is the suggestion that we just require 4.2+ everywhere and not even test anything else?

If so, my suggestion would be to leave the matrix style there with just 4.2 for now.  4.4 can be added easily later.  (Don't change back to the single mongo service style.)  And just to make sure... update lib/config.js serverVersion to ">= 4.2" so the system will not even start with an older server.

Reply:
+1 to keeping matrix style in and seting the serverVersion to 4.2+

Reply:
+1, @mattcollier's comment about 4.2-only is addressed by PR #40, marking this thread as resolved.